src/table/components/body/__tests__/withSelections.spec.tsx

```diff
@@ -20,20 +20,26 @@ describe('withSelections', () => {
   const renderWithSelections = () =>
     render(
       <TestWithProviders selectionsAPI={selectionsAPI}>
-        <HOC
-          cell={cell}
-          styling={styling}
-          announce={announce}
-          column={column}
-          areBasicFeaturesEnabled={areBasicFeaturesEnabled}
-        >
-          {value}
-        </HOC>
+        <table>
+          <tbody>
+            <tr>
+              <HOC
+                cell={cell}
+                styling={styling}
+                announce={announce}
+                column={column}
+                areBasicFeaturesEnabled={areBasicFeaturesEnabled}
+              >
+                {value}
+              </HOC>
+            </tr>
+          </tbody>
+        </table>
       </TestWithProviders>
     );
 
   beforeEach(() => {
-    HOC = withSelections.default((props: CellHOCProps) => <TableCell {...props}>{props.children}</TableCell>);
+    HOC = withSelections((props: CellHOCProps) => <TableCell>{props.children}</TableCell>);
```

Comment:
It might seem odd that I have do not pass props here anymore. But it was never validated in any way so all it did was cause errors to be logged.

Reply:
yeah these tests are a bit strange anyway